test/spec/unit/bootstraps/liveblogTest.js

```diff
@@ -140,7 +140,6 @@ define([
             it('sets up scroll event listener', function () {
                 liveblog.init();
 
-                expect(window.addEventListener).to.have.been.calledOnce;
```

Comment:
Thanks for adding it. I was thinking you could have an expecatation that creativeInjectorMock. trackLiveBlogEpic is called when a new liveblog block is added and more liveblog blocks are loaded. eg. we check whether twitterMock.checkForTweets has been called.

expect(twitterMock.checkForTweets).to.have.been.calledOnce;

We don't need to check anything beyond this as that's the responsibility of the creativeInjector module and would sit in the unit tests for that module...